dev/com.ibm.jbatch.container/src/com/ibm/jbatch/container/jsl/impl/JobModelSerializerImpl.java

```diff
@@ -41,17 +41,7 @@
 
     @Override
     public String serializeModel(JSLJob model) {
-
```

Comment:
so i guess we had this protected but it is no longer necessary to protect this section?